runtime/bcverify/vrfyhelp.c

```diff
@@ -759,7 +776,8 @@ isClassCompatibleByName(J9BytecodeVerificationData *verifyData, UDATA sourceClas
 	if (*targetClassName == '[') {
 		index = parseObjectOrArrayName(verifyData, targetClassName);
 	} else {
-		index = convertClassNameToStackMapType(verifyData, targetClassName, (U_16)targetClassNameLength, 0, 0);
+		UDATA type = IS_QTYPE(*targetClassName) ? BCV_PRIMITIVE_VALUETYPE : BCV_OBJECT_OR_ARRAY;
```

Comment:
We typically dont use `? : ` outside of macros

You can add:

`#define BCV_GET_TYPE_FROM_CHAR(char) ...`

Reply:
addressed this and your other comments in d010fce